conf/defaults.ini

```diff
@@ -1261,19 +1261,22 @@ clean_upgrade = false
 
 # Enable the use of the configured remote Alertmanager and disable the internal one.
 # The default value is `false`.
```

Comment:
You're probably testing but I don't think there should be changes on this file - let's aim for not documenting these fields in the `ini` files for now - opt for documenting them in the settings struct.